nucypher/blockchain/eth/actors.py

```diff
@@ -656,3 +697,342 @@ def request_evaluation(self, evidence):
         receipt = self.adjudicator_agent.evaluate_cfrag(evidence=evidence,
                                                         sender_address=self.checksum_address)
         return receipt
+
+
+class StakeHolder(BaseConfiguration):
+
+    _NAME = 'stakeholder'
+    TRANSACTION_GAS = {}
+
+    class NoFundingAccount(BaseConfiguration.ConfigurationError):
+        pass
+
+    class NoStakes(BaseConfiguration.ConfigurationError):
+        pass
+
+    def __init__(self,
+                 blockchain: BlockchainInterface,
+                 funding_account: str,
+                 funding_password: str = None,
+                 offline_mode: bool = False,
+                 sync_now: bool = True,
+                 *args, **kwargs):
+
+        super().__init__(*args, **kwargs)
+
+        self.log = Logger(f"stakeholder")
+        self.blockchain = blockchain
+        self.offline_mode = offline_mode
+        self.eth_funding = Web3.toWei('0.1', 'ether')  # TODO: How much ETH to use while funding new stakes.
```

Comment:
this should be a constant/cli arg/envvar right?

Reply:
Yes to all. Not sure what makes sense as a default.  Perhaps an amount based on the duration or another way to estimate future gas consumption?

Reply:
well that's all the more reason to make it modifiable without editing the code eh?

Reply:
Indeed.  I want to bring this forth for discussion before implementing it.  If there's not already an open Issue, lets make one.

Reply:
Loosely Related to #842